### PR TITLE
Change package.json so changed to elixir will be non-breaking changes

### PR DESCRIPTION
**Pull request by lukeb**
Previous default version was "*"

### CODE REVIEW

**Comment by brunowego** [comment]
+1

File: package.json [hunk 1]
```diff
@@ -2,6 +2,6 @@
   "private": true,
   "devDependencies": {
     "gulp": "^3.8.8",
-    "laravel-elixir": "*"
+    "laravel-elixir": "~1.0.0"
```
Comment (GrahamCampbell) [inline package.json]: Why not `^1.0.0`?

Comment (lukeb) [inline package.json]: Per http://browsenpm.org/package.json 

^1.0.0 would  >= 1.0.0  and < 2.0.0    (could include breaking changes)
whereas
~1.0.0 would be >= 1.0.0 and < 1.1   (should only be patches, not breaking changes)

"The hope here, is that the package author has followed best-practices and patch versions of the package will NOT be breaking." - taken from the above referenced guide.

Comment (GrahamCampbell) [inline package.json]: "Best practices" would mean minor versions arn't breaking either...

Comment (lukeb) [inline package.json]: This is true. Are you recommending I modify my pull request, then?

Comment (GrahamCampbell) [inline package.json]: Not sure. Ping @taylorotwell and @JeffreyWay regarding how elixer will be versioned.

Comment (JeffreyWay) [inline package.json]: ^1.0.0.

Thanks for the PR.

Comment (lukeb) [inline package.json]: Changed. Ready for merge.

**Comment by rafaell-lycan** [comment]
+1

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**